package.json

```diff
@@ -44,14 +44,15 @@
     "@shopify/stylelint-plugin": "^11.0.0",
     "@shopify/typescript-configs": "^5.1.0",
     "@size-limit/preset-small-lib": "^5.0.3",
-    "@types/jest": "^27.5.1",
+    "@types/jest": "^29.5.12",
     "@types/node": "^20.10.0",
     "babel-loader": "^9.1.2",
     "eslint": "^8.3.0",
     "execa": "^5.0.0",
-    "jest": "^27.5.1",
+    "jest": "^29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
     "jest-preset-stylelint": "^5.0.3",
```

Comment:
There is a newer version of this (`7.0.0`), but it started throwing the following error:

```
TypeError: A dynamic import callback was invoked without --experimental-vm-modules
       at loadLint (../node_modules/.pnpm/jest-preset-stylelint@7.0.0_jest@29.7.0/node_modules/jest-preset-stylelint/getTestRule.js:16:31)
```

Our current version is fully compatible with jest 29, so I'll just leave that upgrade for a different PR.